.markdownlint.json

```diff
@@ -39,5 +39,12 @@
     "MD041": false,
     "MD046": {
         "style": "fenced"
+    },
+    "proper-names": {
+        "code_blocks": false,
+        "names": [
+            "ASP.NET",
+            "PowerShell"
```

Comment:
I didn't add the other names as they have false positives.
If you want to completely remove this from markdownlint and take the fixes only, let me know.